### PR TITLE
Settings Page

### PR DESCRIPTION
**Pull request by ghost**
this include UI, API and Backend for updating and reseting the settings

### CODE REVIEW

**Comment by ahmedqarmout** [comment]
I need to apply the settings of editing names, email and password at the API level to block the users from editing their profile using ajax requests/bypassing the UI

**Comment by esteeroramy** [comment]
Screenshots please @SergeyGV

**Comment by esteeroramy** [comment]
Did you apply the theme colours to the UI? I don't see it in the code

**Comment by SergeyGV** [comment]
Pictures per request

compressed
open

**Comment by esteeroramy** [comment]
Let's include mobile view screenshots as well please

File: Templates/users/users-add.pug [hunk 1]
```diff
@@ -14,7 +14,7 @@ block content
             .col.s12
                 .card.center
                     .card-image
-                        a.btn-floating.halfway-fab.waves-effect.waves-light.red.left(href='/users')
+                        a.btn-floating.halfway-fab.waves-effect.waves-light.cancelButtonBackground-colour.left(href='/users')
```
Comment (esteeroramy) [inline Templates/users/users-add.pug]: Missing text template

File: Templates/users/users-import-complete.pug [hunk 2]
```diff
@@ -2,7 +2,7 @@
     .col.s12
         .card.center
             .card-image
-                a.btn-floating.halfway-fab.waves-effect.waves-light.red.left(href='/users')
+                a.btn-floating.halfway-fab.waves-effect.waves-light.cancelButtonBackground-colour.left(href='/users')
```
Comment (esteeroramy) [inline Templates/users/users-import-complete.pug]: Missing text template

File: Templates/users/users-import.pug [hunk 3]
```diff
@@ -15,7 +15,7 @@ block content
             .col.s12
                 .card.center
                     .card-image
-                        a.btn-floating.halfway-fab.waves-effect.waves-light.red.left(href='/users')
+                        a.btn-floating.halfway-fab.waves-effect.waves-light.cancelButtonBackground-colour.left(href='/users')
                             i.material-icons arrow_back
```
Comment (esteeroramy) [inline Templates/users/users-import.pug]: Missing text template

File: Templates/users/users-edit.pug [hunk 4]
```diff
@@ -14,7 +14,7 @@ block content
             .col.s12
                 .card.center
                     .card-image
-                        a.btn-floating.halfway-fab.waves-effect.waves-light.red.left(href='/users')
+                        a.btn-floating.halfway-fab.waves-effect.waves-light.cancelButtonBackground-colour.left(href='/users')
```
Comment (esteeroramy) [inline Templates/users/users-edit.pug]: Missing text template

**Comment by SergeyGV** [comment]
compressed_m
full_m
users_m

**Review by esteeroramy — APPROVED** [review]
Good work boys

**Pull request merged**